### ISSUE TITLE
no support for pointer arrays

### ISSUE THREAD

**Issue by fleafffb** [awulkiew/graphical-debugging-vscode#3]
it would be nice to have support for allocated memory chunks and have them displayed like in MSVC debugger, for example
ptr, 1024

with float *ptr this will be the same as an array ptr[1024].

**Comment by awulkiew** [comment]
Yes, this feature is missing right now (as you suggested defined as `ptr,size`), as well as visualization of two 1d containers as container of points defined as `container1;container2`.

**Comment by awulkiew** [comment]
No, I'm developing the extension in my free time and I don't have much of it right now. If you don't want to wait or doesn't have time for forking the repo and adding this you could switch from vscode to vscommunity and use this extension: https://marketplace.visualstudio.com/items?itemName=AdamWulkiewicz.GraphicalDebugging

**Comment by fleafffb** [comment]
ah, cool, I didn't know this one. But actually I try to move away from VS in order to get more independent of the platform. If I find the time to dig into VS plugin development I might add a fork. Thanks for your fast responses.

**Comment by awulkiew** [comment]
I'll keep it open because this should still be implemented.

**Comment by awulkiew** [comment]
@fleafffb VS extension has this implemented, though the size has to be a number and cannot be a variable.
The feature you need has to be added in this extension for VSC.
